nixos/modules/programs/teck-programmer.nix

```
@@ -0,0 +1,28 @@
+{ lib, config, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.programs.teck-programmer;
+
+in
+{
+  options.programs.teck-programmer = {
+    enable = mkEnableOption "whether to enable programming the firmware of TECK keyboards";
```

```suggestion
    enable = mkEnableOption "user access to TECK keyboard firmware";
```

`mkEnableOption` generates descriptions that start with "Whether to enable".
https://github.com/NixOS/nixpkgs/blob/b45bd598dcdbc9db50876ee0b01408abf97bc404/lib/options.nix#L97